aiokafka/fetcher.py

```diff
@@ -170,6 +172,12 @@ def __init__(self, client, subscriptions, *, loop,
         self._wait_consume_future = None
         self._wait_empty_future = None
 
+        if type(client.api_version) is tuple and client.api_version >= (0, 10):
```

Comment:
Ok, I checked how ``python-kafka`` handles it, and bam - the same way as here, a shame rly. Well for now let's go with this, will try to poke ppl at python-kafka to do it properly later...